app/controllers/sessions_controller.rb

```diff
@@ -1,33 +1,86 @@
+# app/controllers/sessions_controller.rb
 class SessionsController < ApplicationController
+  before_action :require_login, only: :destroy
   before_action :load_user, only: :create
   before_action :check_authentication, only: :create
+  REMEMBER_ME_CHECKED = "1".freeze
+
+  def new; end
```

Comment:
```suggestion
  // GET: /login
  def new; end
```
E thêm các comment router phía trên các action để clean code hơn nha, tương tự cho những chỗ khác